src/kubectlUtils.ts

```diff
@@ -5,6 +5,8 @@ import { sleep } from "./sleep";
 import { ObjectMeta, KubernetesCollection, DataResource, Namespace, Pod, KubernetesResource } from './kuberesources.objectmodel';
 import { failed } from "./errorable";
 
+export const POD_OUTPUT_PARSING_REGEX = /\s+/g;

```

Comment:
I think I would prefer `KUBECTL_OUTPUT_COLUMN_SEPARATOR` here.  "Parsing regex" is a bit vague as to what it is.